learning-objectives/intl/pt.yml

```diff
@@ -1949,6 +1949,151 @@ ai-prompting/open-ai-api:
+      url: https://www.loom.com/share/65e15aa134c545bc9cfdc6b88bc2ae78?sid=46988f46-bcc0-471f-b89e-55c7ffe953b5
+    - title: Tipos de JOIN em SQL
+      url: https://www.youtube.com/watch?v=HDBdAIcqJQo
+    - title: Diferença entre VIEW e CREATE TABLE
+      url: https://www.loom.com/share/97dbe237b9de459baa4d788f8bb44502?sid=e2168d4f-a8ff-44e4-b3f0-55e72596350a
+    - title: Como utilizar comando WITH e para que serve
+      url: https://www.loom.com/share/8e2f6f70db214e2484af45375539d8e9?sid=222c7e6d-1ff5-430a-ab28-64c025f34ebf
+
+data-analytics/data-exploration:
+  title: Realizar uma análise exploratória
+  description: Realizar uma análise exploratória
+  links:
+    - title: Conceito de variáveis categóricas
+      url: https://docs.google.com/document/d/1BIqa7xR0JNadM9kQExypo3plnwrNia92zCuxkPuMj5A/edit?usp=sharing
+    - title: Versão em áudio
+      url: https://drive.google.com/file/d/13gQ6Ni9FUfDmi5bjQivjeKj04fAI8Gl2/view?usp=drive_link
+    - title: Vídeo sobre tipos de variáveis
+      url: https://www.loom.com/share/afa56686810e46309761bc889364b2d9?sid=cabf587c-7ba1-4419-b501-32c98a73e191
+    - title: Como começar com o Looker Studio
+      url: https://www.youtube.com/watch?v=P5WLaeSQK7s&t=3s
+    - title: Vídeo Como conectar dados no Looker Studio
+      url: https://www.loom.com/share/0cce98d5bbc7482ea421444e2f850cc1
+    - title: Vídeo visão geral do Looker Studio
+      url: https://www.loom.com/share/9c7505eb3c2a4612bdfb7c9d43c40051
+    - title: Vídeo como criar um painel no Looker Studio
+      url: https://www.youtube.com/watch?v=Sprmb4W82sk
+    - title: Tutorial do Google Como criar um relatório no Looker Studio
+      url: https://support.google.com/looker-studio/answer/6292570?hl=ES#zippy=%2Csecciones-de-este-art%C3%ADculo
+    - title: Como criar uma tabela dinâmica no Looker Studio
+      url: https://www.youtube.com/watch?v=3WJrWfAtgEw
+    - title: Conceito de estatísticas descritivas
+      url: https://docs.google.com/document/d/13xu7cf5LcUN2CE58qwyZoNrhS713Ci0tI9S1odeapig/edit?usp=sharing
+    - title: 🔈 Conceito de estatísticas descritivas (versão em áudio)
+      url: https://drive.google.com/file/d/1v6xdtWci7rYtMKibGowl9OxBArT3CVnI/view?usp=drive_link
+    - title: Conceito de medidas de tendência central
+      url: https://docs.google.com/document/d/1Ly_7zLmUjcczqCkQLQwvDYEGyRLtSNQLVi3w8CIh8sI/edit?usp=sharing
+    - title: 🔈 Conceito de medidas de tendência central (versão em áudio)
+      url: https://drive.google.com/file/d/1viy63m4n8Gn9aYHNuLJpd8RzFrTNnlw2/view?usp=drive_link
+    - title: Como criar gráficos no Looker Studio
+      url: https://www.youtube.com/watch?v=B3hPsl-8u3A
+    - title: Conceito de correlação de Pearson
+      url: https://docs.google.com/document/d/1HvuENDveavTmlRtSGk3Gv3LqSPNbRPWNYgOU8e3CxTM/edit?usp=sharing
+    - title: Como calcular correlação no BigQuery
+      url: https://www.loom.com/share/9bf3e48cf57d48ca9c81c2c19562dedf?sid=c4c8a611-d0ae-4c4f-a2f5-200ce48746f4
+
```

Comment:
Sustituir por:

- title: Agrupar dados por variáveis categóricas
  url: https://docs.google.com/document/d/1cOTKw04u2008G7JyRaYHzSzyWRjJwxLryn8hIxkO2es/edit?usp=sharing
- title: Vídeo sobre tipos de variáveis
  url: https://www.youtube.com/watch?v=N7AbQZA06QU&pp=ygUldmFyacOhdmVpcyBjYXRlZ8OzcmljYXMgZSBudW3DqXJpY2FzIA%3D%3D
- title: Link para acessar o Looker Studio
  url: https://lookerstudio.google.com/navigation/reporting
- title: Vídeo como começar no Looker Studio
  url: https://www.youtube.com/watch?v=Gx5S8Ry54TU&pp=ygUjY29tbyBjcmlhciB1bSBwYWluZWwgbm8gZGF0YSBzdHVkaW8%3D
- title: Vídeo Como conectar dados no Looker Studio
  url: https://www.youtube.com/watch?v=dAZ0yfRSKjQ&pp=ygUiY29tbyBjb25lY3RhciBkYWRvcyBubyBkYXRhIHN0dWRpbw%3D%3D
- title: Visão geral Looker Studio
  url: https://www.youtube.com/watch?v=dfAHZ1C_ZQs&list=PLbnAsJ6zlidug4pEvq6tm0h7DPJGkP-aU
- title: Vídeo como criar um painel/dashboard no Looker Studio
  url: https://www.youtube.com/watch?v=qoKPW5Yn3Dk&pp=ygUjY29tbyBjcmlhciB1bSBwYWluZWwgbm8gZGF0YSBzdHVkaW8%3D
- title: Tutorial do Google Como criar um relatório no Looker Studio
  url: https://support.google.com/looker-studio/answer/06292570?0hl=ES#zippy=%2Csections-of-this-art%C3%ADculo%2Cneste-artigo
- title: Tabelas no Looker Studio
  url: https://www.youtube.com/watch?v=WcxlQG1UdkY
- title: Video como criar gráficos de barras
  url: https://www.youtube.com/watch?v=p4nNJWtStAs
- title: Conceito de estatística descritiva
  url: https://docs.google.com/document/d/12YZ-984bRqWxKGr8M-Yzrq8MjWubBw_i38SEcjDt9tE/edit?usp=sharing
- title: Conceito de medidas de tendência central
  url: https://docs.google.com/document/d/15_eD0wsb1w7GFMmmAm8HPFZJGNzlmsZalMwBke9hSt0/edit?usp=sharing
- title: Conceito de correlação de Pearson
  url: https://docs.google.com/document/d/1z_GT13b-GJphDGujDneULn2IOz-KuSmrQpSJBepJF7o/edit?usp=sharing
- title: Como calcular a correlação no BigQuery
  url: https://www.loom.com/share/447a64fa6bd5431289a5a9ca0014e8cb?sid=1edc546a-9691-400a-8cde-b63a2f395532